rust/src/dcerpc/dcerpc_udp.rs

```diff
@@ -294,9 +294,11 @@ pub unsafe extern "C" fn rs_dcerpc_udp_get_tx_cnt(vtx: *mut std::os::raw::c_void
 /// Probe input to see if it looks like DCERPC.
 fn probe(input: &[u8]) -> (bool, bool) {
     match parser::parse_dcerpc_udp_header(input) {
-        Ok((_, hdr)) => {
+        Ok((leftover_bytes, hdr)) => {
             let is_request = hdr.pkt_type == 0x00;
             let is_dcerpc = hdr.rpc_vers == 0x04 &&
+                hdr.fragnum == 0 &&
```

Comment:
Not sure about this.
From the RFC, `fragnum` is used in case of multi PDU transmission. While we don't do anything about that yet, this just means we're looking at the first fragment in a multi PDU transmission. ref: "Fragment Number" (case sensitive) in https://pubs.opengroup.org/onlinepubs/9629399/chap12.htm#tagcjh_17_01

Reply:
For me, this is good, the probing parser should only run for "first" fragments

Reply:
That's a good point that I overlooked. ok then let's let this be.